book/content/credit/credit.md

```diff
@@ -79,6 +83,50 @@ It's perfectly possible to cite a dataset or software package directly, and most
 [JORS]: https://openresearchsoftware.metajnl.com/
 [JOSS]: https://joss.theoj.org/
 
+## How to cite data?
+
+A dataset citation includes all of the same components as any other citation:
+- Author
+- Title
+- Year of publication
+- Publisher (for data, this is often the data repository where it is housed),
+- Version (if indicated)
+- Access information (a URL or DOI)
+
+**APA citation style:** 
+Author/Rightsholder. (Year). Title of data set (Version number). [Retrieved from https://] ***OR*** [DOI]
+
+See also [this example](https://www.force11.org/node/4771) provided by FORCE11.
+
+### Where to cite data in papers?
+
+You should cite your dataset directly in the paper in places where it is relevant, just like publications, as well as in a **data availability statement** at the end of the paper (similar to the acknowledgement section). You can find examples of these statements in the publishers' (research data) author policies, or below:
+    
+### Data availability statement examples:
+**Using the Digital Object Identifier (DOI):**
+“The data that support the findings of this study are openly available in [repository name] at http://doi.org/[doi].”
+
+**If no DOI is issued:** 
+- “The data that support the findings of this study are openly available in [repository name] at [URL], reference number [reference number].”
+
+**When there is an embargo period you can reserve your DOI and still include a reference to the dataset in your paper:**
+- “The data that support the findings will be available in [repository name] at [URL / DOI] following a [6 month] embargo from the date of publication to allow for commercialisation of research findings.” 
+
+**When data cannot be made available:**
+- “Restrictions apply to the data that support the findings of this study. [Explain nature of restrictions e.g. the data contains information that could compromise the privacy of research participants] Data are available upon reasonable request by contacting [name and contact details] and with permission of [third party name].” 
+-  “The data that support the findings of this study are available upon request. Access conditions and procedures can be found at [URL to restricted access repository such as [EASY](https://easy.dans.knaw.nl/ui/home).]”
+
+## How to make software citeable?
+
```

Comment:
I think it would also be useful to add link to software citation principles https://www.force11.org/software-citation-principles